### PR TITLE
Bump version to 0.7.0

### PR DESCRIPTION
**Pull request by EduPonz**
Signed-off-by: Eduardo Ponz <eduardoponz@eprosima.com>

### CODE REVIEW

File: docs/rst/notes/notes.rst [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 
-This release adds the following **improvement**:
+This release adds the following **feature**:
 
-* Update statistics type support including physical data in `DISCOVERY_TOPIC`
+* Possibility of loading Fast DDS Statistics Backend's :class:`DomainParticipant` from Fast DDS XML configuration files.
+
+This release adds the following **improvements**:
+
+* Warn the user the Fast DDS Statistics module needs to be enable.
```
Comment (MiguelCompany) [inline docs/rst/notes/notes.rst:12]:
```suggestion
* Warn the user the Fast DDS Statistics module needs to be enabled.
```

File: versions.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+Version 0.7.0
+-------------
+
+This release adds the following **feature**:
+
+* Possibility of loading Fast DDS Statistics Backend's `DomainParticipant` from Fast DDS XML configuration files.
+
+This release adds the following **improvements**:
+
+* Warn the user the Fast DDS Statistics module needs to be enable.
```
Comment (MiguelCompany) [inline versions.md:13]:
```suggestion
* Warn the user the Fast DDS Statistics module needs to be enabled.
```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Just two nits

**Review by jsantiago-eProsima — APPROVED** [review]
LGTM

**Pull request merged**